cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -1079,17 +1110,27 @@ subroutine stress (nx_block,   ny_block,   &
       !-----------------------------------------------------------------
       ! viscous coefficients and replacement pressure
       !-----------------------------------------------------------------
-         
-         call viscous_coeffs_and_rep_pressure (strength(i,j), tinyarea(i,j),&
-                                               Deltane,       Deltanw,      &
-                                               Deltasw,       Deltase,      &
-                                               zetax2ne,      zetax2nw,     &
-                                               zetax2sw,      zetax2se,     &
-                                               etax2ne,       etax2nw,      &
-                                               etax2sw,       etax2se,      &
-                                               rep_prsne,     rep_prsnw,    &
-                                               rep_prssw,     rep_prsse,    &
-                                               capping)
+
+         call viscous_coeffs_and_rep_pressure_T (strength(i,j), tinyarea(i,j),&
+                                                 Deltane,       zetax2ne,     &
+                                                 etax2ne,       rep_prsne,    &
+                                                 capping)
+ 
+         call viscous_coeffs_and_rep_pressure_T (strength(i,j), tinyarea(i,j),&
+                                                 Deltanw,       zetax2nw,     &
+                                                 etax2nw,       rep_prsnw,    &
+                                                 capping)
+
+         call viscous_coeffs_and_rep_pressure_T (strength(i,j), tinyarea(i,j),&
+                                                 Deltasw,       zetax2sw,     &
+                                                 etax2sw,       rep_prssw,    &
+                                                 capping)
+
+         call viscous_coeffs_and_rep_pressure_T (strength(i,j), tinyarea(i,j),&
+                                                 Deltase,       zetax2se,     &
+                                                 etax2se,       rep_prsse,    &
+                                                 capping)
+
```

Comment:
That is not strictly necessary, right ? it's just cleaning up the B-grid implementation. So maybe it could be done in a later step ? If it's bit for bit, then all for the better and I guess it does not matter much...